Scripts/Core/Contexts/EditingContextManager.cs

```diff
@@ -109,6 +109,7 @@ static void ReopenOnExitPlaymode()
 			}
 		}
 
+#if UNITY_2017_2_OR_NEWER
```

Comment:
Rather than having the Unity_2017_2_OR_NEWER check for each of these, let's have one that wraps all three functions, and an else that defines empty versions of them

Reply:
We wouldn't need empty versions, since OnEnable and OnDisable aren't explicitly called. I can just do one big one with no else.